README.md

```diff
@@ -91,24 +91,32 @@ To make Kong Admin accessible from other locations, let's setup Kong itself to p
 From the [admin console](#user-content-admin-console):
 ```bash
 # Create the authenticated `/kong-admin` API, targeting the localhost port:
-curl http://localhost:8001/apis -i -X POST \
-  --data name=kong-admin \
-  --data uris=/kong-admin \
-  --data upstream_url=http://localhost:8001 \
-  --data https_only=true \
-  --data http_if_terminated=true
-curl http://localhost:8001/apis/kong-admin/plugins/ -i -X POST \
+curl http://localhost:8001/services/ -i -X POST \
+  --data 'name=kong-admin' \
+  --data 'protocol=http' \
+  --data 'port=8001' \
+  --data 'host=localhost'
+# Note the Service ID returned in previous response, use it in place of `$SERVICE_ID`.
```

Comment:
If you add a line like
`export SERVICE_ID=<copy the service_id from previous response>`
you are able to just copy paste the following commands

Reply:
We don't include `export` commands because they lead to problems with people cut-&-pasting, then not realizing there are side-effects lurking in there command line environment.